internal/handlers/telegram/handler_test.go

```diff
@@ -130,3 +132,122 @@ func TestJoinNoUsername(t *testing.T) {
 	}
 	joinHandler(clientObj, testListUser)
 }
+
+func TestStickerSmile_withusername(t *testing.T) {
```

Comment:
I would like to only use CamelCase in unit tests.

Reply:
fixed